tests/integration/transfer_transaction_e2e_test.py

```
@@ -448,3 +448,78 @@ def test_integration_transfer_transaction_approved_token_transfer():
+        .freeze_with(env.client).sign(new_account_private_key)
+
+        receipt = transfer_tx.execute(env.client)
+
+        assert (
+            receipt.status == ResponseCode.SUCCESS
+        ), f"NFT transfer failed with status: {ResponseCode(receipt.status).name}"
+
+        query_transaction = CryptoGetAccountBalanceQuery(account_id)
+        balance = query_transaction.execute(env.client)
+
+        assert balance and balance.token_balances == {token_id: serial_number}
```

Ruff (0.14.10)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)